kgraphql/src/test/kotlin/com/apurebase/kgraphql/integration/DataLoaderExecutionTest.kt

```diff
@@ -25,7 +29,6 @@ class DataLoaderExecutionTest {
 
         query("items") {
             resolver { amount: Int? ->
-                println("query:items")
                 delay(Random.nextLong(50..250L))
```

Comment:
Off-topic:

Do you understand why we'd want randomness in our tests?

Reply:
No. Maybe it is an attempt to have some out-of-order returns 🤷